RAW/ViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ViewController.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    var posts : [PostModel] = []
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view, typically from a nib.
+        fetchDataWithAPI()
+    }
+    
+    func fetchDataWithAPI(){
+        WebService().fetchRawData(sucess:{[weak self](userData) in
+            DispatchQueue.main.async {[weak self] () -> Void in
```

Comment:
make the closure a single one. Look at how Alamofire does it. It helps you to "bail fast"

Reply:
done